mount.go

```diff
@@ -291,3 +304,52 @@ func addStorages(storages []*pb.Storage, s *sandbox) ([]string, error) {
 
 	return mountList, nil
 }
+
+// refer to https://github.com/kubernetes/kubernetes/blob/v1.12.2/pkg/util/mount/mount_linux.go#L512
```

Comment:
Need to fix CI failure about comment here

Reply:
Thank you for your opinion, I'll fix it.